src/helpers/validations.js

```diff
@@ -1,10 +1,15 @@
 const required = value => (value ? undefined : 'Required');
+
 const minValue = min => value => (value.toString().length >= min ? undefined : `Should be greater than ${min}`);
+
+const emailFormat = value => (!value.match(/^([a-zA-Z0-9_.-])+@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/) ? 'Enter Valid email' : undefined);
```

Comment:
Did you write it yourself, or just copy ? :P

Reply:
I can explain this regex. :P